presto-docs/src/main/sphinx/presto_cpp/properties.rst

```diff
@@ -148,9 +148,9 @@ The configuration properties of Presto C++ workers are described here, in alphab
 * **Type:** ``integer``
 * **Default value:** ``100UL << 30``
 
-  Specifies the max spill bytes limit set for each query. This is used to cap the
-  storage used for spilling. If it is zero, then there is no limit and spilling
-  might exhaust the storage or takes too long to run.
+  Specifies the max spill bytes limit set for each query on a Prestissimo worker.
```

Comment:
```suggestion
  Specifies the max spill bytes limit set for each query on a worker.
```
New suggestion following @tdcmeehan's suggestion in [his comment](https://github.com/prestodb/presto/pull/23541/files#r1738639670).